### ISSUE TITLE
Begin working on IA timeline and goals based on incoming top tasks insights

### ISSUE THREAD

**Issue by bonnieAcameron** [uswds/uswds-team#256]
Expected date to present proposal to team: 4/21 

(may be sooner but this requires a bit of collaboration)

**Comment by bonnieAcameron** [comment]
Pushed back deadline to end of next sprint; We're looping in Amy C and Sarah for their work on the content audit and subpages.

**Comment by bonnieAcameron** [comment]
Blocked by audit completion

**Comment by brunerae** [comment]
Closing as no longer relevant.
